### ISSUE TITLE
binary svn properties not base64-decoded

### ISSUE THREAD

**Issue by stephane-chazelas** [gnustavo/SVN-Look#3]
If a svn property contains certain characters (like control characters or non-UTF8 ones, 0xe9 in my example), the svnlook proplist output looks like

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<properties>
<target
   path="/a">
<property
   encoding="base64"
   name="test2">6XgiCnkK
</property>
</properties>
```

SVN::Look ignores the `encoding="base64"`

And the property in the hash returned by `proplist` contains the base64 encode value.

**Comment by gnustavo** [comment]
Hi Stéphane.

I think this change 84edb1e fixes this problem.

What do you think?

**Comment by gnustavo** [comment]
I've just released version 0.40 which should fix this.
